src/envelope.rs

```diff
@@ -121,7 +123,7 @@ pub(crate) trait BroadcastMessageEnvelope: MessageEnvelope + Sync {
     fn clone(&self) -> Box<dyn BroadcastMessageEnvelope<Actor = Self::Actor>>;
 }
 
-impl<A: Handler<M>, M: Send + 'static + Clone + Sync> BroadcastMessageEnvelope
+impl<A: Handler<M>, M: Send + Sync + Clone + 'static> BroadcastMessageEnvelope
```

Comment:
It is a bit unnecessary, but I reordered them to make it easier to read - spent about 5min before I figured out that there was a Sync bound, since I usually expect them together when they are both specified. Do you think we can keep it like this in this PR?